java/src/jmri/LocoAddress.java

```diff
@@ -25,6 +25,14 @@
 
     public Protocol getProtocol();
 
+    /*
+    * Returns true if it is a long address.
+    * If the protocol doesn't support short and long addresses, always return true.
+    */
+    default public boolean isLongAddress() {
+        return true;
```

Comment:
As @devel-bobm pointed out, this isn't appropriate here. Long address is a DCC concept that doesn't necessarily exist elsewhere.